components/event-publisher-proxy/pkg/receiver/receiver_internal_unit_test.go

```diff
@@ -0,0 +1,29 @@
+//go:build unit
```

Comment:
Please rename all unit test files from `...internal_unit_test.go` to `..._unit_test.go` as discussed here https://github.com/kyma-project/kyma/pull/15481#issuecomment-1248342968.